src/client/dfs/dfs.c

```diff
@@ -2960,6 +2972,8 @@ lookup_rel_path(dfs_t *dfs, dfs_obj_t *root, const char *path, int flags,
 	for (token = strtok_r(rem, "/", &sptr);
 	     token != NULL;
 	     token = strtok_r(NULL, "/", &sptr)) {
+		if (is_root)
+			is_root = false;
```

Comment:
Couldn't we just do this? Is compiler and/or low level instruction smart enough?
```suggestion
		is_root = false;
```